src/auro-flight-main.js

```diff
@@ -189,8 +189,7 @@ class AuroFlightMain extends LitElement {
   }
 }
 
-/* istanbul ignore else */
-// define the name of the custom component
+/* eslint max-statements: ["error", 11] */
```

Comment:
Hi @blackfalcon, do I also need to refactor this file to achieve this eslint requirement?

Reply:
For now, I'm setting this rule to 18 in this [commit](https://github.com/AlaskaAirlines/auro-flight/pull/64/commits/d0cb5d821be0dd0d87840eba0f4b4a63105c5021) for related file only.
Also, please advise if this is not needed. Thanks.